accepted/color-4-new-spaces.md

```diff
@@ -2300,7 +2306,8 @@ ie-hex-str($color)
 
 * If `$color` is not a color, throw an error.
 
-* Let `rgb` be the result of [converting] and [gamut mapping] `$color` to `rgb`.
+* Let `rgb` be the result of [converting] and [gamut mapping] `$color` to `rgb`
```

Comment:
Nit: it feels a little odd that `rgb` here stands for a variable and for a channel

Reply:
It's pretty common here to use the name of a color space to represent "the current color in that space". It is a bit odd but I think not so much that a reader can't get used to it, and it's substantially terser than using something like `rgbColor` everywhere.